myfitnesspal/client.py

```diff
@@ -578,6 +578,9 @@ def set_measurements(
         # gather the IDs for all measurement types
         measurement_ids = self._get_measurement_ids(document)
 
+        # get the authenticity token for this edit
+        authenticity_token = document.xpath("(//form[@action='/measurements/new']/input[@name='authenticity_token']/@value)", smart_strings=False)[0]
```

Comment:
Hi Julian, is the `smart_strings` argument needed here? This seems to be working fine for me without it.

Reply:
i honestly don't remember why this is in here. judging from the documentation, you're right, it shouldn't need to be.